tools/start-local-vm

```diff
@@ -80,6 +83,8 @@ Options:
                         private partition will be lost
     --firmware-code     override the default firmware executable file
     --firmware-vars     override the initial firmware variable storage file
+    --boot-image-size   resize the boot disk image to the given size (e.g. 4096M)
```

Comment:
nit: in `buildsys` we refer to this as the [os image](https://github.com/bottlerocket-os/bottlerocket/blob/baef0849029ba7708033f8b350ef91780d3eff91/tools/buildsys/src/manifest.rs#L144):
```suggestion
    --os-image-size   resize the OS disk image to the given size (e.g. 4096M)
```

Reply:
Good point! Makes a lot of sense to stay consistent here. I added a commit that adapts the terminology for the `boot_image` variable names while I'm at it.